src/main/java/org/craftercms/deployer/impl/rest/TargetController.java

```diff
@@ -395,11 +402,16 @@ public void unlockTarget(@ValidateSecurePathParam @ValidateNoTagsParam @PathVari
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping(DUPLICATE_TARGET_URL)
-    public void duplicateTarget(@ValidateSecurePathParam @ValidateNoTagsParam @PathVariable(ENV_PATH_VAR_NAME) String env,
+    public ResponseEntity<Result> duplicateTarget(@NotEmpty @Size(max = 50) @ValidateSecurePathParam @ValidateNoTagsParam @PathVariable(ENV_PATH_VAR_NAME) String env,
```

Comment:
Magic number?

Reply:
I will move this one (and some others we have in deployer validations) to a constant